### DISCUSSION TITLE
decouple frontend

### DISCUSSION THREAD

**Discussion by PurelandToday** [mediacms-io/mediacms#511] [General]
Not sure if it is possible  now already, if we can decouple the frontend, so I could host the frontend somewhere else like github pages.

I am new to mediacms.

**Comment by Micrufun** [comment]
As far as I understand, the ReatJS front-end is integrated into and used by the MediaCMS `templates`. So, `templates` might be among the 1st things to look at.

**Comment by mgogoulos** [comment]
The frontend could be decoupled and served separately but this would require some heavy refactoring! It's not possible for now
